lte/gateway/python/magma/mobilityd/ip_address_man.py

```diff
@@ -432,18 +432,21 @@ def release_ip_address(
                 )
                 raise IPNotInUseError("IP not found in used list: %s" % ip)
 
-            IP_RELEASED_TOTAL.inc()
+        IP_RELEASED_TOTAL.inc()
```

Comment:
I would suggest extracting the changes to the locking (as well as the "scaling tests") to a second PR. They're unrelated to the publishing of the tool, and this PR is already pretty large.

Reply:
Moved to different PR.